cms/envs/common.py

```diff
@@ -413,6 +416,7 @@
 DEFAULT_FROM_EMAIL = 'registration@example.com'
 DEFAULT_FEEDBACK_EMAIL = 'feedback@example.com'
 SERVER_EMAIL = 'devops@example.com'
+STANFORD_COPYRIGHT_EMAIL = 'copyright@stanfordonline.zendesk.com'
```

Comment:
hmm, lets default this to "copyright@example.com" and then we set it to the Stanford zendesk version in secrets.